README.md

```diff
@@ -0,0 +1,71 @@
+
+# EMC_post
+
+The EMC_post package provides tools for post processing for NCEP models.
+
+This is part of the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS)
+project.
+
+## Authors
+
+NCEP/EMC Developers
+
+Code Manager: WenMeng
+
+## Prerequisites
+
+This package requires the following NCEPLIBS packages:
+
+- [NCEPLIBS-g2](https://github.com/NOAA-EMC/NCEPLIBS-g2)
+- [NCEPLIBS-g2tmpl](https://github.com/NOAA-EMC/NCEPLIBS-g2tmpl)
+- [NCEPLIBS-sp](https://github.com/NOAA-EMC/NCEPLIBS-sp)
+- [NCEPLIBS-ip](https://github.com/NOAA-EMC/NCEPLIBS-ip)
+- [NCEPLIBS-bacio](https://github.com/NOAA-EMC/NCEPLIBS-bacio)
+- [NCEPLIBS-w3emc](https://github.com/NOAA-EMC/NCEPLIBS-w3emc)
+- CRTM
```

Comment:
This is an important one for many of our community users. can we add repo info in some manner here too. Assume it's https://github.com/noaa-emc/emc_crtm?

Reply:
Yes, I will add.